R/remote-info.R

```diff
@@ -0,0 +1,32 @@
+getRemoteInfo <- function(pkgRecord) {
+  if (pkgRecord$source == "github") {
+    remote_info <- as.data.frame(
+      c(
+        list(
+          RemoteType     = pkgRecord$source,
+          GithubRepo     = pkgRecord$gh_repo,
+          GithubUsername = pkgRecord$gh_username,
+          GithubRef      = pkgRecord$gh_ref,
+          GithubSHA1     = pkgRecord$gh_sha1
+        ),
+        c(GithubSubdir   = pkgRecord$gh_subdir)
```

Comment:
Having the subdir fields outside the main list is subtle; maybe add a comment explaining why this approach is necessary.

Reply:
Yes, this is exactly what tripped *me* up last time. And I _knew_ why they were outside the main list. I almost feel like doing this much more verbosely but more explicitly would be better.

Reply:
I tried to explain what's going on.